invenio_rdm_records/records/mappings/v6/rdmrecords/drafts/draft-v1.0.0.json

```diff
@@ -12,20 +12,47 @@
         },
         "access" : {
           "properties" : {
-            "access_right" : {
+            "record" : {
               "type" : "keyword"
             },
-            "created_by" : {
-              "type" : "long"
+            "files" : {
+              "type" : "keyword"
+            },
+            "owned_by" : {
```

Comment:
Comment: I think this is correct, but we have to validate that this structure properly supports the queries we have to make.

Say e.g. we need to query for all records owned by a given user:

``acccess.owned_by.user:1``